query/stdlib/influxdata/influxdb/from.go

```diff
@@ -38,14 +37,6 @@ func init() {
 	flux.ReplacePackageValue("influxdata/influxdb", influxdb.FromKind, flux.FunctionValue(FromKind, createFromOpSpec, fromSignature))
 	flux.RegisterOpSpec(FromKind, newFromOp)
 	plan.RegisterProcedureSpec(FromKind, newFromProcedure, FromKind)
-	plan.RegisterPhysicalRules(
-		FromConversionRule{},
-		MergeFromRangeRule{},
-		MergeFromFilterRule{},
-		FromDistinctRule{},
-		MergeFromGroupRule{},
```

Comment:
Are we just not going to push down group rules? I'm fine with that, but I just want to confirm that's intended.

Reply:
I thought it was incompatible with the new rules? Is that not the case?

Reply:
It is. I just wanted to confirm that, at least temporarily, this was ok. We should create an issue to redo this rule with the new rules also.

Reply:
@nathanielc that is correct, none of the old rules are compatible.
@jsternberg Group will need to be a separate PR.

Reply:
Sounds good, Issue created https://github.com/influxdata/influxdb/issues/13623

Can either of you add more details to that issue if they are needed? Also does that rule need a new RPC call to be exposed from storage?

Reply:
Sure I'll add it. And yes that will be the final RPC call to be exposed from storage.